### PR TITLE
Update G7Sensor.swift

### PR DESCRIPTION
**Pull request by chmloop**
including Dexcom One+

### CODE REVIEW

File: Dependencies/G7SensorKit/G7SensorKit/G7CGMManager/G7Sensor.swift [hunk 1]
```diff
@@ -214,8 +214,8 @@ public final class G7Sensor: G7BluetoothManagerDelegate {
             return .ignore
         }
 
-        /// The Dexcom G7 advertises a peripheral name of "DXCMxx", and later reports a full name of "Dexcomxx"
-        if name.hasPrefix("DXCM") {
+        /// The Dexcom G7 advertises "DXCMxx" & te One+ "DX02xx", and later reports a full name of "Dexcomxx"
+        if name.hasPrefix("DXCM") || name.hasPrefix("DXCM") {
```
Comment (lifeguardz) [inline Dependencies/G7SensorKit/G7SensorKit/G7CGMManager/G7Sensor.swift:218]: typo?
```suggestion
        if name.hasPrefix("DXCM") || name.hasPrefix("DX02") {
```

**Comment by chmloop** [comment]
Hi,

code already modified after my first submit attempt into

if name.hasPrefix("DXCM") || name.hasPrefix("DX02") {

Tx for noticing

**Comment by Jon-b-m** [comment]
What not just using *if name.hasPrefix("DX") {* instead?

**Comment by chmloop** [comment]
Received the One+ yesterday, I'll test it over the weekend

**Comment by chmloop** [comment]
test is running ...

**Comment by Jon-b-m** [comment]
Can you please make a PR to dev branch instead?

**Comment by Jon-b-m** [comment]
OK. I can make a PR to dev. I'll close this. Please next time  always PR to dev

**Pull request closed without merging**